manifests/init.pp

```diff
@@ -33,6 +33,7 @@
   Optional[Enum['on', 'off']] $daemon                        = undef,
   $daemon_user                                               = $::nginx::params::daemon_user,
   $daemon_group                                              = undef,
+  Optional[Array[String]] $dynamic_module                    = undef,
```

Comment:
Can you update it so it defaults to an empty array?

Reply:
I'd also prefer it to be named ```dynamic_modules``` since it's an array.